pages/index.html

```diff
@@ -5,8 +5,46 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <section class = "container">
```

Comment:
Based on HTML styling guide, attributes should not have spaces around the `=` assignment.
```html
  <section class="container">
```